app/src/main/res/raw/okta_app_auth_config.json

```diff
@@ -1,10 +1,11 @@
 {
-  "client_id": "{your-client-id-here}",
-  "redirect_uri": "{your-redirect-uri}",
+  "client_id": "{clientIdValue}",
```

Comment:
just use `{clientId}` (our docs are standardizing on this)

Reply:
done